components/ArtPieces.js

```diff
@@ -0,0 +1,12 @@
+import ArtPiecePreview from "./ArtPiecePreview";
+export default function ArtPieces({ pieces }) {
+  return (
+    <ul>
+      <h1>Art Gallery</h1>
+      {pieces.map((piece) => {
+        const { slug } = piece;
+        return <ArtPiecePreview key={slug} piece={piece} />;
+      })}
+    </ul>
```

Comment:
I would move the heading outside of the unordered list:

```suggestion
<>
<h1>Art Gallery</h1>
<ul>
      {pieces.map((piece) => {
        const { slug } = piece;
        return <ArtPiecePreview key={slug} piece={piece} />;
      })}
</ul>
</>
```